### ISSUE TITLE
how  to install sass 3.3.0?

### ISSUE THREAD

**Issue by luckypoem** [sass/sass#2183]
i use command "gem install sass" to install sass.but its version is 3.4.22:

```
root@AR:~# sass --version
Sass 3.4.22 (Selective Steve)
root@AR:~#
```

now i want to install sass 3.3.0,what should i do?

**Comment by Koshmaar** [comment]
I'd love to install 4.0 , but `gem install sass -v 4.0` gives

> ERROR:  Could not find a valid gem 'sass' (= 4.0) in any repository
> ERROR:  Possible alternatives: sass

Do you know who is responsible for managing ruby sass package?

Btw, I need 4.0 because it fixes the #193

**Comment by nex3** [comment]
@luckypoem You can pass `-v 3.3.0` to install version 3.3.0.

@Koshmaar 4.0 is not yet released. You can install 4.0.0-alpha.1, but it's very old at this point and releasing new versions is not currently a high priority. Your best bet is probably do install from the Git repository.
